src/js/gsoc-guide.js

```
@@ -0,0 +1,388 @@
+      return storageGet('shortlistedOrgs') || [];
+    } catch (_) {
+      return [];
+    }
+  }
+
+  /* ─────────────────────────────────────────────
+   * 8. INPUT SANITISATION  (B7, client-side layer)
+   * ───────────────────────────────────────────── */
+  function sanitiseInput(raw) {
+    return raw
+      .replace(/[\x00-\x1F\x7F]/g, '')  // strip control characters
```

ESLint

[error] Unexpected control character(s) in regular expression: \x00, \x1f.

(no-control-regex)